# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoader.h
# Commit: 7e54a223001d
# Full Hash: 7e54a223001dd1c0057b74c7691dc3821947a3df
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 3
# Description:
#   Bug 1742437 - move DOM script context into own file and rename it; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138132
# ==============================================================================

diff -r 3a34d557d474 -r 7e54a223001d dom/script/ModuleLoader.h
--- a/dom/script/ModuleLoader.h	Thu Feb 17 15:57:51 2022 +0000
+++ b/dom/script/ModuleLoader.h	Thu Feb 17 15:57:51 2022 +0000
@@ -8,6 +8,7 @@
 #define mozilla_dom_ModuleLoader_h
 
 #include "ModuleLoaderBase.h"
+#include "mozilla/dom/ScriptLoadContext.h"
 #include "mozilla/dom/ScriptLoadRequest.h"
 #include "ScriptLoader.h"
 
@@ -59,7 +60,7 @@
   static already_AddRefed<ModuleLoadRequest> CreateTopLevel(
       nsIURI* aURI, ScriptFetchOptions* aFetchOptions,
       const SRIMetadata& aIntegrity, nsIURI* aReferrer, ScriptLoader* aLoader,
-      DOMScriptLoadContext* aContext);
+      ScriptLoadContext* aContext);
 
   // Create a module load request for a static module import.
   already_AddRefed<ModuleLoadRequest> CreateStaticImport(
@@ -68,7 +69,7 @@
   // Create a module load request for dynamic module import.
   static already_AddRefed<ModuleLoadRequest> CreateDynamicImport(
       nsIURI* aURI, ScriptFetchOptions* aFetchOptions, nsIURI* aBaseURL,
-      DOMScriptLoadContext* aContext, ScriptLoader* aLoader,
+      ScriptLoadContext* aContext, ScriptLoader* aLoader,
       JS::Handle<JS::Value> aReferencingPrivate,
       JS::Handle<JSString*> aSpecifier, JS::Handle<JSObject*> aPromise);
 };